_schemaV2.graphql

```diff
@@ -9670,6 +9670,8 @@ type DoNotUseThisPartner {
   subscription_state: String @deprecated(reason: "Use subscriptionState")
 }
 
+union EditMeOrderResponse = OrderMutationError | OrderMutationSuccess
```

Comment:
Coming from Rails conventions, maybe we can name it (and others below) with `Update...` instead of `Edit...`?